lib/editor/util/map.js

```diff
@@ -44,11 +44,14 @@ type R5Response = {
   }>
 }
 
+export const coordIsOutOfBounds = (coords: LatLng, bounds: any) => {
```

Comment:
Are we expecting a Leaflet type here for `bounds` rather than `any`?

Reply:
I copied the type from the `stopIsOutOfBounds` this is adapted from. However `bounds` passes flow check so I've made the change

Reply:
Fixed in 2bae81f17a9497173531d0c7261e7cdc1f2982a2